### PR TITLE
A few improvements: remove hardcoded /var/spool/cwl; expose cpu and memory to be set by user or use sensible defaults

### PR DESCRIPTION
**Pull request by lindaxiang**
Three main features in the PR:
* removed dependence on hardcoded /var/spool/cwl path
* defined addition output files in CWL for individual VCFs
* exposed cpu core and memory to be set by user or use sensible dynamic defaults

### CODE REVIEW

**Comment by junjun-zhang** [comment]
@lindaxiang, do new features implemented in the PR require anything change to the Dockerfile? If not, I think we should revert it back. Not able to build docker image using the original Dockerfile is a different problem, we should address separately.

File: Dockstore.cwl [hunk 1]
```diff
@@ -17,7 +17,7 @@ dct:contributor:
 
 requirements:
 - class: DockerRequirement
-  dockerPull: quay.io/pancancer/pcawg-sanger-cgp-workflow:2.0.3
+  dockerPull: quay.io/pancancer/pcawg-sanger-cgp-workflow:expose-cpu-memory
```
Comment (denis-yuen) [inline Dockstore.cwl:20]: Is this correct? 
Maybe something like `quay.io/pancancer/pcawg-sanger-cgp-workflow:2.0.3-expose-cpu-memory` would make it easier to tell where this falls in terms of versions

Comment (junjun-zhang) [reply Dockstore.cwl:20]: Good question, this should be a general thing, this sounds like a good practice we should follow.

Just to be clear, currently, we just make docker image tag to match git repo branch name. When formally making a release, we will need to update docker image tag to match the release tag (which is actually done by Quay automatically).

At this moment, docker image build fails due to likely dependency problems, the Dockerfile is pretty old. This is a separate issue, I am thinking to propose something to deal with it as this is happening to other PCAWG workflows, and likely will happen in the future.

**Comment by junjun-zhang** [comment]
Received confirmation from @keiranmraine via email:
> I've had a quick look and if it's working I can't see anything surprising in the code.

**Review by junjun-zhang — APPROVED** [review]

**Pull request merged**